webapp/components/admin_console/rate_settings.jsx

```diff
@@ -138,7 +138,7 @@ export default class RateSettings extends AdminSettings {
                     label={
                         <FormattedMessage
                             id='admin.rate.httpHeaderTitle'
-                            defaultMessage='Vary By HTTP Header:'
+                            defaultMessage='Limit by HTTP Header:'
```

Comment:
Check with dev on this and update help text? Not clear what this does and how it should be named.